internal/worker/firewaller/firewaller.go

```diff
@@ -109,14 +53,18 @@ type Config struct {
 
 	CredentialAPI common.CredentialAPI
 
-	// TODO: (jack-w-shaw) Drop these once we move tests to mocks based
+	// These are used to coordinate gomock tests.
+
 	// WatchMachineNotify is called when the Firewaller starts watching the
 	// machine with the given tag (manual machines aren't watched). This
 	// should only be used for testing.
 	WatchMachineNotify func(tag names.MachineTag)
 	// FlushModelNotify is called when the Firewaller flushes it's model.
-	// This should only be used for testing
+	// This should only be used for testing.
 	FlushModelNotify func()
+	// FlushMachineNotify is called when the Firewaller flushes a machine.
+	// This should only be used for testing.
```

Comment:
Hi Adis, perhaps the line here about using only for test should be removed since we use this func in the below func (fw *Firewaller) flushMachine(machined *machineData)?